doc/src/Developer_org.rst

```diff
@@ -102,42 +103,44 @@ LAMMPS while passing it the command line flags and input script. It
+be passed to the constructor. All constructors for classes derived from
+it, must do so and thus pass that pointer to the constructor for
+:cpp:class:`LAMMPS_NS::Pointers`.  The default constructor for
+:cpp:class:`LAMMPS_NS::Pointers` is disabled to enforce this.
 
 Since all storage is supposed to be encapsulated (there are a few
 exceptions), the LAMMPS class can also be instantiated multiple times by
-a calling code.  Outside of the aforementioned exceptions, those LAMMPS
+a calling code.  Outside the aforementioned exceptions, those LAMMPS
 instances can be used alternately.  As of the time of this writing
-(early 2021) LAMMPS is not yet sufficiently thread-safe for concurrent
+(early 2023) LAMMPS is not yet sufficiently thread-safe for concurrent
 execution.  When running in parallel with MPI, care has to be taken,
 that suitable copies of communicators are used to not create conflicts
 between different instances.
 
-The LAMMPS class currently (early 2021) holds instances of 19 classes
-representing the core functionality.  There are a handful of virtual
-parent classes in LAMMPS that define what LAMMPS calls ``styles``.  They
-are shaded red in the :ref:`class-topology` figure.  Each of these are
-parents of a number of child classes that implement the interface
-defined by the parent class.  There are two main categories of these
-``styles``: some may only have one instance active at a time (e.g. atom,
-pair, bond, angle, dihedral, improper, kspace, comm) and there is a
-dedicated pointer variable for each of them in the composite class.
+The LAMMPS class currently holds instances of 19 classes representing
+the core functionality.  There are a handful of virtual parent classes
+in LAMMPS that define what LAMMPS calls ``styles``.  These are shaded
+red in the :ref:`class-topology` figure.  Each of these are parents of a
+number of child classes that implement the interface defined by the
+parent class.  There are two main categories of these ``styles``: some
+may only have one instance active at a time (e.g. atom, pair, bond,
+angle, dihedral, improper, kspace, comm) and there is a dedicated
+pointer variable for each of them in the corresponding composite class.
 Setups that require a mix of different such styles have to use a
-*hybrid* class that takes the place of the one allowed instance and then
-manages and forwards calls to the corresponding sub-styles for the
-designated subset of atoms or data.  The composite class may also have
-lists of class instances, e.g. Modify handles lists of compute and fix
-styles, while Output handles a list of dump class instances.
+*hybrid* class instance that acts as a proxy, and manages and forwards
+calls to the corresponding sub-style class instances for the designated
+subset of atoms or data.  The composite class may also have lists of
+class instances, e.g. ``Modify`` handles lists of compute and fix
+styles, while ``Output`` handles a list of dump class instances.
```

Comment:
Should we make all LAMMPS class names ``inline literal``?

Reply:
In an ideal world, those would be documented with Doxygen comments and their API would be imported into the Developer's Guide reference section and then those would be referred to with ":cpp:class:`Modify <LAMMPS_NS::Modify>`" . But we are still a long way away from that. When I see it, I use "``" markup for files, class names/functions, and defines or CMake variables. To make them stand out more. But I don't see this as a "must have" change.

Reply:
We have arrived at over 2800 pages (in PDF format) in the manual, so there is a lot of "legacy text" around that was added before we started to pay closer attention to consistency.  Keep in mind that most files were auto-converted to .rst markup from txt2html markup and that had its issues with some corner cases (even of we have by now eliminated 99.99% of those, at that ratio and with 2800 pages, there would still be 28 pages with a leftover issue)